x-pack/plugins/lens/public/datasources/form_based/dimension_panel/reference_editor.tsx

```diff
@@ -82,7 +82,6 @@ export interface ReferenceEditorProps {
   labelAppend?: EuiFormRowProps['labelAppend'];
   isFullscreen: boolean;
   toggleFullscreen: () => void;
-  setIsCloseable: (isCloseable: boolean) => void;
```

Comment:
I removed all code associated with it because seems like this is not needed anymore... I am not sure what changed it but it was introduced 2 years ago with formula editor PR (https://github.com/elastic/kibana/pull/99297). The comment says it solves ' Popover steals focus from text input, and doesn't close when clicking the monaco editor. Needs an alternative design', but I couldn't reproduce it and tested quite extensively.